tka/tka.go

```diff
@@ -39,8 +39,15 @@ type Authority struct {
 	head           AUM
 	oldestAncestor AUM
 	state          State
+}
 
-	storage Chonk
+// Clone duplicates the Authority structure.
+func (a *Authority) Clone() Authority {
```

Comment:
Would it be better to return a `*Authority`? I thought that was the usual convention but I'm unclear on if there is a meaningful distinction in this case.

Reply:
Done.